Assets/Scripts/Player/PlayerHealth.cs

```diff
@@ -21,6 +22,15 @@ public void TakeDamage(int damage)
             KillPlayer();
         }
     }
+
+    public void Heal(int heal)
+    {
+        if (health < maxHealth)
```

Comment:
this 2 if statements could be reduced to a simple if else statement  if(health+heal>maxhealth)health=maxHealth;
else health+=heal;